src/assets/lang/ru.json

```diff
@@ -291,7 +291,8 @@
     "title": "Журнал здоровья",
     "symptom": "Симптом",
     "symptoms": "симптом",
-    "symptoms_plural": "симптома (-ов)",
+    "symptoms_1": "симптома",
```

Comment:
Is this intentional?

Reply:
Yes:

> This also fixes i18next pluralisation in Haitian and Russian, which I discovered weren't working properly while fixing pluralisation in Yiddish.

Russian didn't work before because i18next ignores `_plural` and instead looks for these keys:

- `_0` (default) which it uses for any number ending in `1` (singular)
- `_1` which it uses for numbers ending `2`, `3`, `4` (genitive)
- `_2` which it used for other numbers (plural)

I double-checked that the two  and if we plural forms the translator put in do go this way round for both cases.

It's a bit of a pain that Russian needs different keys but it's pretty unlikely we'd change these.

Reply:
FYI, if we support Arabic we'll need something similar for it as Arabic has about 5 different forms like this; but there are only two translation lines affected so it's not a big burden, just another item for the checklist.